web-console/public/demo/publicDemos.json

```diff
@@ -0,0 +1 @@
+../../../demo/publicDemos.json
```

Comment:
why is it called public?

Reply:
I was looking for a way to name .json with the list demos that are available through the WebConsole UI - I decided on `publicDemos.json`, let me know if you have other ideas
This file specifically is a symlink to the original in demo/

Reply:
I'd just call it demos.json. Also we should avoid putting symlinks in git -- it will not work anymore on windows.